pgmpy/tests/test_utils/test_utils.py

```diff
@@ -51,6 +51,28 @@ def test_get_example_model(self):
             m = get_example_model(model=model)
             del m
 
+    def test_get_example_model_dagitty(self):
+        all_models = [
+            "M-bias",
+            "confounding",
+            "mediator",
+            "paths",
+            "Sebastiani",
+            "Polzer",
+            "Schipf",
+            "Shrier",
+            "Acid",
+            "Thoemmes",
+            "Kampen",
+            "Didelez",
```

Comment:
I would suggest keeping the year with the names here. Those models were taken from some papers, so would make it easier to look up which paper it was exactly.